README.md

```diff
@@ -2104,6 +2105,29 @@ Only necessary properties can be passed while modifying the config. All other pr
 
 Delete operation is also available for configurations, either standard or factory ones.
 
+#### Controlling workflows
+
+[OsgiFramework](src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/osgi/OsgiFramework.kt) allows to use `workflowManager` object to manage workflows in custom tasks.
+Workflows can be either enabled or disabled by calling a corresponding method, with name(s) provided as a parameter:
+```kotlin
+workflowManager.disable("update_asset_create")
+workflowManager.disable(listOf("update_asset_create", "update_asset_mod"))
+```
+
+Additionally, framework provides a method which allows to disable or enable workflows only for particular set of tasks:
+```kotlin
+register("disableWorkflowsWhileSomethingHappens") {
```

Comment:
this indicates that task is not solving real case ;> avoid that to make your example looking more serious